packages/plugin-conversation/src/interceptor.js

```diff
@@ -0,0 +1,106 @@
+/**!
+ *
+ * Copyright (c) 2015-2016 Cisco Systems, Inc. See LICENSE file.
+ * @private
+ */
+
+import {Interceptor} from '@ciscospark/http-core';
+import {isArray} from 'lodash';
+
+/**
+ * Encrypts, Normalizes, and Decrypts conversation service payloads
+ */
+export default class ConversationInterceptor extends Interceptor {
+  /**
+   * @returns {EncryptionInterceptor}
+   */
+  static create() {
+    return new ConversationInterceptor({spark: this});
+  }
+
+  /**
+   * @see Interceptor#onRequest
+   * @param {Object} options
+   * @returns {Promise<Object>}
+   */
+  onRequest(options) {
+    if (!options.method || options.method.toUpperCase() === `GET`) {
+      return options;
+    }
+
+    return this.spark.device.isSpecificService(`conversation`, options.service || options.uri)
```

Comment:
Nice. So for each new plugin that needs to be encrypted we'll just make a new interceptor that's similar to this one.